python/Ganga/Core/MonitoringComponent/Local_GangaMC_Service.py

```diff
@@ -1158,11 +1158,15 @@ def makeCredChecker(self, credObj):
         def credChecker():
             log.debug("Checking %s." % getName(credObj))
             try:
-                s = credObj.renew()
-            except Exception as msg:
+                credObj.renew()
+            except CredentialRenewalError:
```

Comment:
It's at `Ganga.GPIDev.Credentials2.exceptions.CredentialRenewalError`. I'm unsure whether it should be move to be alongside the core exceptions, I feel like it's violating the modularisation a bit?